x/ibc/05-port/keeper/keeper.go

```diff
@@ -12,12 +12,14 @@ import (
 // Keeper defines the IBC connection keeper
 type Keeper struct {
 	scopedKeeper capability.ScopedKeeper
+	router       map[string]types.IBCModule
```

Comment:
map from portID to callbacks

Reply:
Hmm, how will this be kept up-to-date when the chain restarts?

Reply:
Can we delete this now?